style.css

```diff
@@ -1,23 +1,31 @@
 body {
   display: flex;
-  background: url(background.png);
-  background-repeat: no-repeat;
-  background-size: cover;
-  position: relative;
+  flex-direction: column;
+  padding: 0%;
+  margin: 0%;
+  background-color: #e5e5e5;
 }
 
 .wrapper {
   max-width: 100%;
+  height: 85vh;
   display: flex;
   flex-direction: column;
+  background: url(img/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom-left-radius: 5rem;
+  background-color: #fff;
 }
 
 .header {
   display: flex;
```

Comment:
- [ ] [OPTIONAL] I think it's good to avoid repeating the same lines multiple times in a code. You can wrap it in a class and use the class whenever it's needed in the HTML code. for example : 
`.flex {
display: flex;
}`  
you can do the same to any repeated code, it will make your code DRY.